apollo-backend/src/main/resources/application.conf

```diff
@@ -19,4 +19,9 @@ apollo {
     port = 8082
     idleTimeoutSeconds = 300
   }
+
+  deployment {
+    //30 minutes * 60 = 1800 seconds
+    forceCancelTimeoutSeconds = 1800
+  }
 }
```

Comment:
Other conf files don't contain a new line at the end of the file

Reply:
some of them do contain newline (docker/apollo.conf)
It could prevent from parsing and rendering issues when working with this file in Linux
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
cool! good to know :)